nautobot/ipam/querysets.py

```diff
@@ -405,6 +405,12 @@ def get_or_create(self, defaults=None, **kwargs):
         namespace = kwargs.pop("namespace", None)
         host = kwargs.get("host")
         mask_length = kwargs.get("mask_length")
+        address = kwargs.get("address")
+        if host is None and isinstance(address, str):
+            address = netaddr.IPNetwork(address)
+            host = str(address.ip)
+            mask_length = address.prefixlen
```

Comment:
Maybe?

```suggestion
        if host is None:
            if isinstance(address, str):
                address = netaddr.IPNetwork(address)
            host = str(address.ip)
            mask_length = address.prefixlen
```

Reply:
I was thinking maybe so, but unsure. Does the queryset `address` filter support IPNetwork objects?

Reply:
Should we change this to

```py
if host is None and address is not None:
    address = netaddr.IPNetwork(address)
```